x-pack/plugins/endpoint/server/handlers/endpoint_handler.ts

```diff
@@ -67,6 +68,13 @@ export class EndpointHandler implements EndpointRequestContext {
             sort: [{ created_at: 'desc' }],
           },
         },
+        aggs: {
```

Comment:
Can you send me a sample of the response back to the front end. Would like to see where to get the total from if different from the current `hits.total.value`

Reply:
```"hits" : {
    "total" : {
      "value" : 300,
      "relation" : "eq"
    },
"aggregations" : {
    "total" : {
      "value" : 100
    }
  }```